gadgets/snapshot_process/artifacthub-pkg.yml

```diff
@@ -0,0 +1,32 @@
+# Artifact Hub package metadata file
+# This file has the automatically generated from gadget.yaml
+version: 0.1.0
+name: "snapshot process"
+category: monitoring-logging
+displayName: "snapshot process"
+createdAt: "2024-03-13T21:13:29+01:00"
+description: "Show running processes"
+logoURL: "https://inspektor-gadget.io/media/brand-icon.svg"
+license: ""
+homeURL: "https://inspektor-gadget.io/"
+containersImages:
+    - name: gadget
+      image: "ghcr.io/inspektor-gadget/gadget/snapshot_process:latest"
+      platforms:
+        - linux/amd64
+        - linux/arm64
+keywords:
+    - gadget
+links:
+    - name: source
+      url: "https://github.com/inspektor-gadget/inspektor-gadget/"
+install: |
+    # Run
+    ```bash
+    sudo ig run ghcr.io/inspektor-gadget/gadget/snapshot_process:latest
```

Comment:
Currently we still have to enable the experimental mode. But I don't know if we should add it in every yml and remove it after

Reply:
I think it's best to be consistent and add it everywhere, otherwise users will be confused why it does not work.

Removing it everywhere later should be easy with `sed` or equivalent.